Dockerfile

```
@@ -1,3 +1,15 @@
+FROM golang:buster as build
+
+WORKDIR /app/
+
+# Install dependencies
+COPY go.mod go.sum ./
+RUN go mod download
+
+COPY . .
+
+RUN make
```

```suggestion
RUN make build
```
just to not rely on the default target